training.md

```diff
@@ -86,10 +86,10 @@ This section is divided into sort filters, sort of. See each section and find ou
+ - http://cryto.net/~joepie91/blog/2015/07/19/why-you-should-never-ever-ever-use-mongodb/ - why you should never, ever, ever use MongoDB. NOTE: read that until you are sure you will never use it. If you are still not convinced after reading it, read [this](https://blog.scrapinghub.com/2013/05/13/mongo-bad-for-scraped-data/), if you are still not convinced, try [this](http://www.sarahmei.com/blog/2013/11/11/why-you-should-never-use-mongodb/comment-page-1/), if you want more recent info, try [this](https://www.mongodb.com/community/forums/t/primary-node-in-replica-set-down-and-2-weeks-of-data-lost/110472/6), and if you still are not entirely sure, you might want to know that their "atomic" property actually makes it so that if you have a collection with some objects, and you'll query to get 1000 objects based on some criteria during an update operation, you will sometimes get 999 back. And that is by design, they closed the bug as "won't fix". So if you ask mongodb for something, you have to check if it returned what you asked for, because if it didn't, then you need to retry and *if you are lucky* then eventually you'll get what you asked for. Not everyone likes to write code like that and most people want to have access to their data always, not just when it's their lucky day. And also [this](https://www.infoq.com/news/2014/04/bitcoin-banking-mongodb/). If the largest cryptomarkets couldn't use it properly, are you sure you will be able to? :) Anyway, if you are still not sure whether you can use it or not, speak up on `#default`, someone will help you;
```

Comment:
```suggestion
 - http://cryto.net/~joepie91/blog/2015/07/19/why-you-should-never-ever-ever-use-mongodb/ - why you should never, ever, ever use MongoDB. NOTE: read that until you are sure you will never use it. If you are still not convinced after reading it, read [this](https://blog.scrapinghub.com/2013/05/13/mongo-bad-for-scraped-data/), if you are still not convinced, try [this](http://www.sarahmei.com/blog/2013/11/11/why-you-should-never-use-mongodb/comment-page-1/), if you want more recent info, try [this](https://www.mongodb.com/community/forums/t/primary-node-in-replica-set-down-and-2-weeks-of-data-lost/110472/6 - TL;DR: in a cluster overwhelmed with writes during primary node failure most writes landed in a journal but became inaccessible after failback and in practice lost because there is no way to retrieve it from there. June 2021.), and if you still are not entirely sure, you might want to know that their "atomic" property actually makes it so that if you have a collection with some objects, and you'll query to get 1000 objects based on some criteria during an update operation, you will sometimes get 999 back. And that is by design, they closed the bug as "won't fix". So if you ask mongodb for something, you have to check if it returned what you asked for, because if it didn't, then you need to retry and *if you are lucky* then eventually you'll get what you asked for. Not everyone likes to write code like that and most people want to have access to their data always, not just when it's their lucky day. And also [this](https://www.infoq.com/news/2014/04/bitcoin-banking-mongodb/) - if the largest cryptomarkets couldn't use it properly, are you sure you will be able to? :) Anyway, if you are still not sure whether you can use it or not, speak up on `#default`, someone will help you;
```

Reply:
but "failback" was the correct word here. You don't think it's appropriate?

Reply:
I have to admit that I didn't know the word (and neither did my built-in dictionary). After checking the meaning, I have to agree. Fixed.